packages/input-iban/test/validators.test.js

```diff
@@ -19,4 +19,14 @@ describe('IBAN validation', () => {
     expect(nlValidator.execute('DE89370400440532013000')).to.be.true;
     expect(nlValidator.execute('foo')).to.be.true;
   });
+
+  it('provides IsNotCountryIBAN to prevent IBANs from specific countries', () => {
+    const nlValidator = new IsNotCountryIBAN('NL');
+    const deValidator = new IsNotCountryIBAN('DE');
+    expect(nlValidator.execute('NL17INGB0002822608')).to.be.true;
+    expect(deValidator.execute('DE89370400440532013000')).to.be.true;
+    expect(nlValidator.execute('DE89370400440532013000')).to.be.false;
+    expect(deValidator.execute('NL17INGB0002822608')).to.be.false;
+    expect(nlValidator.execute('foo')).to.be.false;
```

Comment:
I think test is failing on this line. Which is expected. The IsNotCountryIBAN does not fail on "foo", but the IsIBAN validator that is applied by default on input-iban, does fail. I think this line can be removed, since it's not a test for the IsNotCountryIBAN, but more of a input-iban/IsIBAN specific test.